src/oscar/templatetags/wishlist_tags.py

```diff
@@ -3,36 +3,10 @@
 register = template.Library()
 
 
-@register.tag(name="wishlists_containing_product")
-def do_basket_form(parse, token):
+@register.simple_tag
+def wishlists_containing_product(wishlists, product):
     """
     Template tag for adding the user's wishlists form to the
     template context so it can be rendered.
```

Comment:
Please remove the bit in the docstring suggesting it adds the variable to the template context (which was an implementation detail of the previous tag), and just say it returns the value? Or just remove the whole thing.